text/chapter2.md

```diff
@@ -18,23 +18,20 @@ The rest of the chapter will guide you through installing and configuring these
 
 ## Installing PureScript
 
-The recommended approach to installing the PureScript compiler is to download a binary release for your platform from the [PureScript website](https://www.purescript.org).
+You'll need [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node) and to be [able to install global packages](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory) to proceed.
```

Comment:
The 'install global packages' link provides two options for fixing the EACCES error. The second option's first step is to "backup your computer." While that may be true, it might discourage people from installing any further.
Could we change the rendering of the first link to say something like this?

> You will need to install `Node.js` and `npm`. We recommend installing [Node.js and npm via a "node version manager"](https://docs.npmjs.com/getting-started/installing-node) to avoid issues with installing packages gloablly. If you choose to install it manually, you might experience the [`EACCES` error when installing packages globally](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory).

Reply:
Looks good. Applied your version. I'm an nvm fan myself.